src/main/java/input/InformationInput.java

```diff
@@ -0,0 +1,11 @@
+package input;
+
+public class InformationInput {
```

Comment:
InformationInput이 Console.readLine()까지 하도록 하면 
```java
        informationInput.printInformationInputOfCarName();

        String carName = Console.readLine();
```
이 부분을 하나의 함수를 호출하는 것으로 해결할 수 있을 것 같아요!

Reply:
> InformationInput이 Console.readLine()까지 하도록 하면
> 
> ```java
>         informationInput.printInformationInputOfCarName();
> 
>         String carName = Console.readLine();
> ```
> 
> 이 부분을 하나의 함수를 호출하는 것으로 해결할 수 있을 것 같아요!

그렇군요! 제가 따로 나눈 이유는 informationInput은 사용자가 input 내리기 전 알리는 정보성 메세지를 보낸다라는 클래스를 만든 것이기 때문에, 분리한 것입니다! 읽는데 불편함이 있으셨을까요? 아니면 좀더 깔끔한 코드를 위하여 그렇게 말씀하신걸까요?

Reply:
좀 더 깔끔한 코드가 나올 수 있다고 생각되어서 입니다!
그리고 저번 주에 저도 InputView에서 안내 문구 출력 + Console.readLine()을 함께 하는 것과 InputView에서는 Console.readLine()만 하고 OutputView에서 안내 문구 출력을 하도록 하는 것 중에 고민했었는데,
그 때 피드백을 남겨주셨던 대부분의 분들이 안내 문구 출력은 입력과 연관되어 있는 것이니 InputView로 옮기는 것이 좋겠다는 의견을 주셨습니다!
그래서 함께 합치면 좋을 것 같다고 생각했습니당

Reply:
> 좀 더 깔끔한 코드가 나올 수 있다고 생각되어서 입니다! 그리고 저번 주에 저도 InputView에서 안내 문구 출력 + Console.readLine()을 함께 하는 것과 InputView에서는 Console.readLine()만 하고 OutputView에서 안내 문구 출력을 하도록 하는 것 중에 고민했었는데, 그 때 피드백을 남겨주셨던 대부분의 분들이 안내 문구 출력은 입력과 연관되어 있는 것이니 InputView로 옮기는 것이 좋겠다는 의견을 주셨습니다! 그래서 함께 합치면 좋을 것 같다고 생각했습니당

음 그럴 수도 있을 것 같네요. 연관되어 있는 부분이고, 사실 작디 작은 부분이라 InputView로 옮기는 편이 나을 것 같네요. 해당 기능이 어딘가에 들어갈 수 있는 작은 기능인지, 아니면 큰 기능인지 파악하는 것도 저의 중요한 과제가 되겠네요! 감사합니다!

Reply:
의견 잘 남겨주셔서 저도 제 의견을 남깁니다.
메소드 를 묶는 단위는 이 두개가 따로 있을때 이상하지 않은가? 를 기점으로 생각하는거 같아요.

극단적으로, 
`System.out.("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")` 출력 부분에 대한 입력과

`System.out.println("시도할 횟수는 몇 회인가요?");` 출력 부분에 대한 입력이 서로 바뀌어져서 받으면 어떻게 되나요?

이런 관점에서 묶어보는걸 시도하는걸 추천합니다.

물론, 한라인의 메소드를 묶으라는건 아닙니다. ( 싱크홀 패턴 주의 )

Reply:
> 의견 잘 남겨주셔서 저도 제 의견을 남깁니다. 메소드 를 묶는 단위는 이 두개가 따로 있을때 이상하지 않은가? 를 기점으로 생각하는거 같아요.
> 
> 극단적으로, `System.out.("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")` 출력 부분에 대한 입력과
> 
> `System.out.println("시도할 횟수는 몇 회인가요?");` 출력 부분에 대한 입력이 서로 바뀌어져서 받으면 어떻게 되나요?
> 
> 이런 관점에서 묶어보는걸 시도하는걸 추천합니다.
> 
> 물론, 한라인의 메소드를 묶으라는건 아닙니다. ( 싱크홀 패턴 주의 )

> 의견 잘 남겨주셔서 저도 제 의견을 남깁니다. 메소드 를 묶는 단위는 이 두개가 따로 있을때 이상하지 않은가? 를 기점으로 생각하는거 같아요.
> 
> 극단적으로, `System.out.("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")` 출력 부분에 대한 입력과
> 
> `System.out.println("시도할 횟수는 몇 회인가요?");` 출력 부분에 대한 입력이 서로 바뀌어져서 받으면 어떻게 되나요?
> 
> 이런 관점에서 묶어보는걸 시도하는걸 추천합니다.
> 
> 물론, 한라인의 메소드를 묶으라는건 아닙니다. ( 싱크홀 패턴 주의 )

역시 프로그래머마다 기준이 각양각색이네요 🥇 

그렇다면 영수님 같은 경우 
```suggestion
informationInput.printInformationInputOfCarName();
String carName = Console.readLine();
```
해당 두 개의 코드가 같이 있을 때 이상하지 않다고 느끼시는걸까요?

Reply:
네. 메소드 명이 이상하게 느껴지긴 하는데, 출력과 입력이 두개 같이 있는건 크게 이상하지 않습니다.

애초에 입력이 무조건 입력만 받는다는 생각을 안하는거 같아요 저는.
입력을 위한 출력이라고 생각하면 더 편하실거 같네요. 
( 단순히 System.out.println(xx) 하나 찍는다고 출력이 아닌, 자동차 결과를 `출력`한다. 가 제가 생각하는 출력 입니다. )

Reply:
> 네. 메소드 명이 이상하게 느껴지긴 하는데, 출력과 입력이 두개 같이 있는건 크게 이상하지 않습니다.
> 
> 애초에 입력이 무조건 입력만 받는다는 생각을 안하는거 같아요 저는. 입력을 위한 출력이라고 생각하면 더 편하실거 같네요. ( 단순히 System.out.println(xx) 하나 찍는다고 출력이 아닌, 자동차 결과를 `출력`한다. 가 제가 생각하는 출력 입니다. )

입력을 위한 출력이므로 입력에 속한다! 라는 말씀도 일리가 있네요! 단순하게 접근하는 것이 아닌 좀더 본질적인 차원에서 접근해봐야겠네요! 좋은 말씀 감사합니다 👍